### PR TITLE
Running endpoints without Cloud Service emulator

### LINKED ISSUE #1583: Azure Host - how to run Cloud Service based endpoint without cloud service emulator
Documentation doesn't contain anything about running CS-hosted endpoint locally w/o CS emulator (by using Azure Host Process). 

With `IConfigureThisEndpoint` moved into hosts (both Azure Host & NSB Host), users that used NSB Host to run endpoints w/o CS emulator will run into a [problem](https://github.com/Particular/NServiceBus/issues/3808). Documentation will help to resolve this by pointing out what to do.

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #1583

### CODE REVIEW

**Comment by SeanFeldman** [comment]
@Particular/docs-maintainers please review
(smoked and tested)

File: nservicebus/hosting/cloud-services-host/running-without-emulator.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Running locally without Azure Cloud Services emulator
+summary: Debugging endpoints without Azure Cloud Services emulator
+component: CloudServicesHost
+redirects:
+ - nservicebus/hosting/cloud-services-host
```
Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: doesnt this redirect from a parent index page?

File: nservicebus/hosting/cloud-services-host/running-without-emulator.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: should this be added to the menu?

Comment (SeanFeldman) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: Good catch. Fixed.

File: nservicebus/hosting/cloud-services-host/running-without-emulator_running-locally_cloudserviceshost_[7,).partial.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ * Add NuGet reference to the NServiceBus.Hosting.Azure.HostProcess package
+ * Update endpoint project Debug start action to point to `NServiceBus.Hosting.Azure.HostProcess.exe`
```
Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator_running-locally_cloudserviceshost_[7,).partial.md]: doesnt this happen with the nuget install?

File: nservicebus/hosting/cloud-services-host/running-without-emulator.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+title: Running locally without Azure Cloud Services emulator
+summary: Debugging endpoints without Azure Cloud Services emulator
+component: CloudServicesHost
+reviewed: 2017-02-21
+---
+
+Debugging endpoints hosted with Cloud Services requires spinning up Azure Cloud Service emulator and loading endpoint in the emulator. It's a heavy operation and if no scale out of roles is required, can be done by loading endpoints using a host process provided by NServiceBus instead.
```
Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: "spinning up" perhaps "starting up"?

Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: "heavy operation" perhaps  "resource intensive"

File: nservicebus/hosting/cloud-services-host/running-without-emulator_running-locally_cloudserviceshost_[7,).partial.md (hunk 3 above)
Comment (SeanFeldman) [inline nservicebus/hosting/cloud-services-host/running-without-emulator_running-locally_cloudserviceshost_[7,).partial.md]: Not that I know

File: nservicebus/hosting/cloud-services-host/running-without-emulator.md (hunk 4 above)
Comment (SimonCropp) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: isnt "NServiceBus" ambiguous? no host is provided in the NServiceBus nuget. not sure how to re-word though

Comment (SeanFeldman) [inline nservicebus/hosting/cloud-services-host/running-without-emulator.md]: Me neither. I'll keep it this way for now.

**Comment by SeanFeldman** [comment]
@SimonCropp updated menu and text. Please review. Thanks

**Pull request merged**